napalm/base/mock.py

```diff
@@ -33,7 +33,7 @@ def raise_exception(result):
 
 
 def is_mocked_method(method):
-    mocked_methods = []
+    mocked_methods = ['traceroute', 'ping']
```

Comment:
black is complaining, I think it is because of the single quotes. You can just run it yourself to double check and/or to have it format it automatically

Reply:
sorry about that.